src/main/scala/slack/api/SlackApiClient.scala

```diff
@@ -39,6 +39,8 @@ object SlackApiClient {
     None
   }
 
+  private[this] val toStrictTimeout = config.getInt("api.tostrict.timeout").seconds
```

Comment:
Can we have a default in the code (e.g. the 10 seconds), so that we provide continuity for people who don't edit their `conf` files?

Reply:
By default typesafe config will read the value in `reference.conf`, which is already set to 10 seconds. Do you think it's better to allow the user to set the timeout programmatically instead?

Reply:
What if someone has their own `reference.conf` without this value? Will it work? Will Akka still read our `reference.conf`?

Reply:
It will work. Relevant typesafe config documentation is [here](https://github.com/lightbend/config#standard-behavior):

In summary:

1. Applications can provide their `application.conf` or `reference.conf`, which will be merged into `reference.conf` provided by the library, overriding the values in the latter.
2. If the application provides a `reference.conf` (or `application.conf`) without this value, it will use the value in the `reference.conf` provided by the library. I verified it with my own application; it worked fine :)

Reply:
Great. Thanks for checking!